ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/AEMDownloadManger.java

```diff
@@ -193,7 +194,11 @@ void syncStaleAemImportsTask() {
      */
     @WorkerThread
     void syncAemImportTask(@NonNull final Uri baseUri, final boolean force) {
-        // TODO
```

Comment:
I was not sure how to handle the force param.  It seems to be handle in the calling Task method.

Reply:
This method may be called on an AEM import we already synced once, force indicates if we should sync no matter what. if `force` is false, then we want to observe `AemImport.isStale()` to determine if we should actually sync.